### PR TITLE
fix: user get balance

### LINKED ISSUE #49: getUserBalance is not a generic ERC20 metadata
The user's balance is user centric metadata. It should be used outside the ERC20 Metadata. 

https://github.com/aurora-is-near/rainbow-bridge-client/blob/f1039b490da07915b94b84056b22c00ff3aef01b/packages/nep141-erc20/src/natural-erc20/getMetadata.js#L72

### PR DESCRIPTION
**Pull request by 0x3bfc**
closes #49

### CODE REVIEW

**Review by mfornet — APPROVED** [review]
LGTM, can you address failing CI

**Comment by 0x3bfc** [comment]
> can you address failing CI

Yeah sure

**Comment by paouvrard** [comment]
Should we move `getBalance` to `getBalance.js` then since it is not metadata ? and rename `getErc20Data` to `getMetadata`?
For some context, I think `getErc20Data` grouped everything together as a convenience to the rainbow-bridge-frontend, https://github.com/aurora-is-near/rainbow-bridge-frontend/blob/06ca162327e16288b389b8ee28843a48d22edecd/src/js/utils.js#L18 but since `getAllowance` was also added separately, it makes send to refactor it.

**Review by paouvrard — APPROVED** [review]
LGTM thank you for refactoring! Please squash your commits to remove the CI fixing. About the commit format, `fix: Commit msg...` is for code patches which require a patch release and would appear as a bug fix in the changelog. For commits which don't require a new release version, `build, chore, ci, docs, refactor...` can be used. https://www.conventionalcommits.org/en/v1.0.0/

File: packages/nep141-erc20/tsconfig.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "extends": "../../tsconfig.json",
-  "include": ["src/*"],
+  "include": ["src/*", "src/natural-erc20/getBalance.js"],
```
[the remark this answers is not in the document]
Comment (0x3bfc) [reply packages/nep141-erc20/tsconfig.json:3]: Yes, will update it 👍

**Comment by paouvrard** [comment]
I think we can close this PR because the library is now using `ethers` and I will be making this refactor in this PR: https://github.com/aurora-is-near/rainbow-bridge-client/pull/58

**Comment by 0x3bfc** [comment]
> I think we can close this PR because the library is now using ethers and I will be making this refactor in this PR: #58

Please close it if you feel that it doesn't add to main branch or the refactor you have in the PR #58 :)

**Pull request closed without merging**